### ISSUE TITLE
resolve NOTE on R build checks v1.3.6

### ISSUE THREAD

**Issue by jread-usgs** [USGS-R/hazardItems#58]
```
* checking R code for possible problems ... NOTE
Setting endpoint to  https://marine.usgs.gov/coastalchangehazardsportal/ 
```

**Comment by mhines-usgs** [comment]
this has to do with auth and setting the prod environment in setBaseURL.R

    if (endpoint=="prod"){
        pkg.env$url_base = "https://marine.usgs.gov/coastalchangehazardsportal/"
        cat('Setting endpoint to ',pkg.env$url_base,'\n')
    }

Need to address a few endpoint needs before we can resolve this. Will create another issue.
